js/main.js

```diff
@@ -2,16 +2,19 @@ import {createOffers} from './data.js';
 
 import {createPopup} from './popup.js';
 
-import {disableСreateForm, disableFilterForm, enableСreateForm, enableFilterForm} from './state.js';
+import {disableAdForm, disableFilterForm, enableAdForm, enableFilterForm} from './state.js';
+
+import './ad-form.js';
 
 const map = document.querySelector('.map__canvas');
 
 const offers = createOffers();
 
 offers.forEach((offer) => map.append(createPopup(offer)));
 
-disableСreateForm();
-disableFilterForm();
+disableAdForm ();
+disableFilterForm ();
 
-enableСreateForm();
-enableFilterForm();
+enableAdForm ();
```

Comment:
Т.е. имеет смысл инициализировать валидацию только когда принято решение разблокировать форму, например где-то тут. (Потом изменишь и будешь вызывать это в нужном месте, передавать доп аргументы и прочее)
```suggestion
enableAdForm ();
initValidation();
```